### ISSUE TITLE
[Enhancement] Support the Live Tree element selection and display of adorners on iOS and Android too

### ISSUE THREAD

**Issue by andreinitescu** [dotnet/maui#1138]
## Live Tree element tools: The element selection and the display of adorners

The **Live visual tree** features, like the **element selection** and the **display of adorners**, are truly great tools for productivity, which obviously is one of the most critical aspects of any technology.

Unfortunately these tools are only supported on **UWP**, not also on the much more popular platforms like **iOS** and **Android**:
https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/live-visual-tree#live-visual-tree-toolbar

### MAUI.UWP-only does not help much

MAUI (and Xamarin.Forms) is the .NET truly cross-platform way to reach the popular mobile platforms, like Android or iOS, or both.  If someone wants to target only the UWP platform, you'd just use the actual UWP SDK for Windows. The element selection and display of adorners already exist in the UWP development.

### Why element selection and the display of adorners are important

When developing, it takes a good mental effort and good knowledge of the framework element and behavior, to quickly mentally map between the code you see in the IDE and the actual running mobile screen and viceversa. Sure, this is expected, it's part of the complexity of developing software, but these two tools can really help a lot by making it easier to find the relationship between the code and the rendered UI.

There are several reasons why it can be difficult to mentally map between the XAML code in the IDE and the screen in the running app:

- Sometimes, it's the XAML code which is just too long to read and understand, because it lacks the good practices (componentization, use of common styles, not setting property default values, etc.). 
- Sometimes, the XAML code is long because the UI is just more complex, and there's really not much which can be done to simplify it. 
- It's not easy at all for the junior developers which are new to the framework (don't know well the syntax/purpose/behavior of every of the UI element used in the code), or new to the UI development in general, or even just new to coding 
- Actually, there are parts in the XAML itself which are just too verbose (no intention to criticize, I don't think much can be done to avoid the verbosity)

Bottom line is, just like with any other language and code, it really takes good practice and experience to get a really good, clean,  XAML code, and I think these tools would help a lot with the development and maintenance of the MAUI 🏝 apps.

Thank you!

**Comment by hartez** [comment]
Work is happening on something like this right now - https://github.com/dotnet/maui/issues/572

**Comment by jsuarezruiz** [comment]
This is definitely happening!. To avoid duplicates, I am going to close this request to concentrate all the feedback in https://github.com/dotnet/maui/issues/1138

Thanks for all your feedback!

**Comment by jsuarezruiz** [comment]
Can see a demo here https://www.youtube.com/watch?v=566YhzjfN4Y
